src/runner/task.js

```diff
@@ -18,7 +18,7 @@ export default class Task extends AsyncEventEmitter {
         this.tests                   = tests;
         this.opts                    = opts;
         this.proxy                   = proxy;
-        this.screenshots             = new Screenshots(this.opts.screenshotPath, this.opts.screenshotPathPattern, this.opts.screenshotsFullPage);
+        this.screenshots             = new Screenshots(this.opts.screenshots, this.opts);
```

Comment:
It's a strange construction - `new Screenshots(this.opts.screenshots, this.opts);`
First, you pass to the constructor a part of options, then - the whole amount of options.
Simplify to `new Screenshots(this.opts);` or something else.

Reply:
the `Screenshots` constructor will not require `this.opts` parameter in future. This strange construction is only for backward compatibility, so we will definitely remove this in the future

Reply:
Ok